js-sdk/providing_static_data.mdx

```diff
@@ -141,8 +141,21 @@ const tolgee = Tolgee()
 
 `fallbackOnFail` option will tell Tolgee to look for data in the next source, so it will fetch your local data. **Order of plugins is important here, they are used in the same order as they were added.**
 
-:::info Plugins are used in the same order as they were added.
+### Fallback logic
+
+Plugins are used in the same order as they were added.
 Tolgee first checks if data aren't already in cache (data passed directly to `staticData`) then all the plugins and then dynamic data from `staticData` field.
```

Comment:
I know this was not edited in this PR, but maybe it can be more clear when we described like this.:

```
Tolgee tries to find required data in this order:

1. Checks if the data are in the cache
2. Uses the plugins in the order as they are added in the code using the `use()` method
3. Calls the async functions provided using static data.
```

...Example config with all options and description how it will be called...

Reply:
Ok